management-service/services/experiment_service.go

```diff
@@ -153,10 +141,10 @@ func (svc *experimentService) ListExperiments(
 			// * the end_time parameter should fall within the experiment's (start and end) times
 			// * the experiment starts and ends within the [start_time and end_time) duration
 			query = query.Where(
-				`tstzrange(start_time, end_time, '[)') @> tstzrange(?, ?, '[]')
-				OR tstzrange(start_time, end_time, '()') @> tstzrange(?, ?, '[]')
-				OR tstzrange(?, ?, '[]') @> tstzrange(start_time, end_time, '[)')`,
-				params.StartTime, params.StartTime, params.EndTime, params.EndTime, params.StartTime, params.EndTime,
+				svc.query().
```

Comment:
Refactoring this OR clause to use the APIs rather than raw query string, since the new version of the Gorm API supports it correctly.